firmware/user/modes/mode_mazerf.c

```diff
@@ -1156,13 +1156,13 @@ void ICACHE_FLASH_ATTR mzGameoverLedDisplay(void)
             danceTimers[4].timerFn(NULL);
             break;
         case HARD_LEVEL:
-            danceTimers[12].timerFn(NULL);
+            danceTimers[11].timerFn(NULL);
```

Comment:
See line 102 and 106 in mode_dance.c is this a mistake? For dance 8 ... need to subtract 2 to get index. I havent tested my change. Ideally as the levels get harder, the light reward gets more interesting